freedomofpress/securedrop
# Revert "Add instructions for working around NoScript XSS upload problem"

Resolves #4160. 

This reverts commit b35530831db8bcb191f2c8553b1f05fa7e6eab6d (and the dependent commit babc2f52da21c0b8d15eb4bef3c1b6c66b37e744).

## Status

Ready for review

## Test plan

1. Build updated staging environment
2. Connect to source interface using Tor browser version 8.0.8 or greater
3. Verify that "Scan uploads for potential cross-site requests" option (under "Advanced" NoScript settings) is not set. It should be unset by default.
3. Verify that NoScript instructions implemented in #4160 no longer appear on source interface
4. Upload a test file of at least 75MB size repeatedly (I recommend a total of 5 times)

Expected result:
- All uploads complete successfully.

## Deployment

No special considerations. Tor browser includes the required fix as of [version 8.0.8](https://blog.torproject.org/new-release-tor-browser-808), which was also [shipped with Tails 3.13.1](https://tails.boum.org/news/version_3.13.1/index.en.html), and which should auto-updated on most systems.

## Checklist
